content/en/case-studies/adidas/index.html

```diff
@@ -0,0 +1,108 @@
+---
+title: adidas Case Study
+linkTitle: adidas
+case_study_styles: true
+cid: caseStudies
+css: /css/case-studies-gradient.css
+featured: false
+---
+​
+
+
+<div class="article image overlay" style="background-image: url('/images/Adidas1.png')">
+  <h1> CASE STUDY: adidas</h1>
+  <div class="subhead">Staying True to Its Culture, adidas Got 40% of Its Most Impactful Systems Running on Kubernetes in a Year</div>
+</div>
+​
+​
+<div class="details">
+    Company &nbsp;<b>adidas</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Location &nbsp;<b>Herzogenaurach, Germany</b>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Industry &nbsp;<b>Fashion</b>
+</div>
+​
+<hr>
+<section class="section1">
+<div class="colsgradient">
+  <div class="col1">
+    <h2>Challenge</h2>
+    In recent years, the adidas team was happy with its software choices from a technology perspective—but accessing all of the tools was a problem. For instance, “just to get a developer VM, you had to send a request form, give the purpose, give the title of the project, who’s responsible, give the internal cost center a call so that they can do recharges,” says Daniel Eichten, Senior Director of Platform Engineering. “The best case is you got your machine in half an hour. Worst case is half a week or sometimes even a week.”
+    <h2>Solution</h2>
+    To improve the process, “we started from the developer point of view,” and looked for ways to shorten the time it took to get a project up and running and into the adidas infrastructure, says Senior Director of Platform Engineering Fernando Cornago. They found the solution with containerization, agile development, continuous delivery, and a cloud native platform that includes Kubernetes and Prometheus.
```

Comment:
I don't think I understand the punctuation (especially the quote marks here). What is a direct quote, what is reported speed, and what is paraphrasing?

Reply:
The words within the quote marks are the direct quote and what's outside the quote marks are paraphrasing -- all attributed to Fernando within that sentence. Not sure what you mean by "reported speed."